modules/engageyaBidAdapter.js

```diff
@@ -0,0 +1,144 @@
+import {
+  BANNER,
+  NATIVE
+} from '../src/mediaTypes.js';
+
+const {
+  registerBidder
+} = require('../src/adapters/bidderFactory.js');
+const BIDDER_CODE = 'engageya';
+const ENDPOINT_URL = 'https://recs.engageya.com/rec-api/getrecs.json';
+const ENDPOINT_METHOD = 'GET';
+
+function getPageUrl() {
+  var pUrl = window.location.href;
+  if (isInIframe()) {
+    pUrl = document.referrer ? document.referrer : pUrl;
+  }
+  pUrl = encodeURIComponent(pUrl);
+  return pUrl;
+}
+
+function isInIframe() {
+  try {
+    var isInIframe = (window.self !== window.top);
+  } catch (e) {
+    isInIframe = true;
+  }
+  return isInIframe;
+}
+
+export const _getUrlVars = function(url) {
```

Comment:
Hi @reemeng , since you're using this function only in unit tests, don't you think that defining it in the _spec file would be more appropriate (given that you don't use it here at all)?

Reply:
Indeed @aleksatr ,
Will change it and commit

Thanks,
Reem

Reply:
did it :)